datastore/tests/system/test_system.py

```diff
@@ -442,6 +443,17 @@ def test_transaction_via_with_statement(self):
         self.case_entities_to_delete.append(retrieved_entity)
         self.assertEqual(retrieved_entity, entity)
 
+    def test_readwrite_put(self):
```

Comment:
IMO this is an unnecessary test. Transactions are read-write by default and we already have several tests which exercise transactions.